HW1/task4.py

```diff
@@ -0,0 +1,15 @@
+'''Задача 8: Требуется определить, можно ли от шоколадки размером n
+× m долек отломить k долек, если разрешается сделать один разлом по
+прямой между дольками (то есть разломить шоколадку на два
+прямоугольника).'''
+
+print('Введите размер шоколадки n × m')
+n, m = int(input()), int(input())
+print('Cколько долек нужно отломить')
+k = int(input())
```

Comment:
```suggestion
k = int(input('Cколько долек нужно отломить: '))
```
На мой взгляд, так будет немного понятней.